.github/workflows/ci.yml

```diff
@@ -8,6 +8,7 @@ on:
 
 jobs:
   build-libs-linux-amd64:
+    name: build-libs-linux-amd64 😁
```

Comment:
Oh yeah, i am a too arid person to add emoji there :) 
Thank you, we **needed** these! ahha